app/templates/app/create_board.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'app/base.html' %}
+{% block content %}
+    <h2>Create Board</h2>
+    <form method="post" novalidate>
+        {% csrf_token %}
+        {% for forms in form %}
```

Comment:
doesn't make sense, make sure that your variable naming understandable, (loop forms in form?)

Reply:
Done sir